service/worker/deletenamespace/activities.go

```diff
@@ -99,6 +109,35 @@ func (a *localActivities) ValidateProtectedNamespacesActivity(_ context.Context,
 	return nil
 }
 
+func (a *localActivities) ValidateNexusEndpointsActivity(ctx context.Context, nsID namespace.ID, nsName namespace.Name) error {
+	if !a.allowDeleteNamespaceIfNexusEndpointTarget() {
```

Comment:
readability:

if a.allowDeleteNamespaceIfNexusEndpointTarget() { return nil}
...

less nesting, easier to read.